medical_patient_disease_allergy/__manifest__.py

```diff
@@ -1,24 +1,29 @@
 # -*- coding: utf-8 -*-
-# © 2016 LasLabs Inc.
+# Copyright 2016 LasLabs Inc.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Medical Patient Allergies",
     "summary": "Isolate the allergy concept in medical_disease",
-    "version": "10.0.2.0.0",
+    "summary": "Isolates allergies from diseases.",
```

Comment:
I did this in an amend of the original commit, because it was an eff up in my conflict resolution and should not be part of the migration commit